aiohttp_security/jwt_identity.py

```diff
@@ -0,0 +1,30 @@
+"""Identity policy for storing info in the jwt token.
+
+"""
+
+from .abc import AbstractIdentityPolicy
+import jwt
+
+
+class JWTIdentityPolicy(AbstractIdentityPolicy):
+    def __init__(self, secret, algorithm=None, param_name=None):
+        self.secret = secret
+        self.algorithm = 'HS256' if algorithm is None else algorithm
+        self.param_name = 'Authorization' if param_name is None else param_name
+
+    async def identify(self, request):
+        header_identity = request.headers.get(self.param_name)
+        identity = jwt.decode(header_identity,
+                              self.secret,
+                              algorithm=self.algorithm)
+
+        return identity['identity']
+
+    async def remember(self, request, response, identity, **kwargs):
+        encoded_identity = jwt.encode({'identity': identity},
```

Comment:
@asvetlov  Yes, you are right: `remember` is not totally appropriate name for this action. But if I correctly understand, jwt authentication approach doesn't imply for storing jwt token (excluding storing it on client side). For this action the most suitable name would be `login` which provides jwt token in the result or 403 otherwise. But for now we should to implement `AbstractIdentityPolicy` and use the same approach for eachone identity way.

Reply:
Let's keep `remember()`/`forget()` empty.
`login` is a client method, the client should acquire JWT by making a custom HTTP request to some resource. After that he is responsible for providing `Authorization` header properly.